Makefile

```diff
@@ -42,6 +44,10 @@ babelify: dist
 	@$(BABEL) lib/util --out-dir dist/lib/util
 	@$(BABEL) test --out-dir dist/test
 
+node_modules: package.json
```

Comment:
What is the effect of having `package.json` here? There's no Make target for `package.json`.